lib/mp4/transmuxer.js

```diff
@@ -73,6 +73,7 @@ AudioSegmentStream = function(track) {
         // If this is an allowed frame, keep it and record it's Dts
         if (currentFrame.dts >= earliestAllowedDts) {
           track.minSegmentDts = Math.min(track.minSegmentDts, currentFrame.dts);
+          track.minSegmentPts = track.minSegmentDts;
```

Comment:
Might be worth commenting that this is safe because audio frames won't be out of order.